python/app/g2panel

```diff
@@ -345,14 +344,26 @@ class MainWindow(QtWidgets.QMainWindow):
     
     #Setting initial frames
     def setFrames(self):
+        #If trigger mode is selected then the frame number is replaced by the triggers other wise it takes the frame number
+        if self.det.timing==timingMode.TRIGGER_EXPOSURE:
+            pass
         #number of frames needs to go to both receiver and detector
-        self.rcv.frames = self.frameSpinBox.value()
-        self.det.frames = self.frameSpinBox.value()
+        else:
+            self.rcv.frames = self.frameSpinBox.value()
+            self.det.frames = self.frameSpinBox.value()
 
     #Setting initial trigger
     def setTrigger(self):
+        
+        #If trigger mode is selected then the frame number is replaced by the triggers other wise it takes the frame number
+        if self.det.timing==timingMode.TRIGGER_EXPOSURE:
+            self.rcv.frames = self.triggerSpinBox.value()
+            self.det.frames = self.triggerSpinBox.value()
```

Comment:
No need to set frames when number of triggers change.

Reply:
Removed frames